Mathlib/Logic/Godel/PartArith.lean

```diff
@@ -0,0 +1,363 @@
+/-
+Copyright (c) 2024 Shogo Saito. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Shogo Saito. Adapted for mathlib by Hunter Monroe
```

Comment:
I think you can just do
```suggestion
Authors: Shogo Saito, Hunter Monroe
```
The point of this line is to know who to complain if something breaks 😉